release/ray_release/tests/test_run_script.py

```diff
@@ -1,108 +1,133 @@
 import json
 import os
-import shutil
 import subprocess
 import sys
 import tempfile
-import unittest
+import pytest
 
 from ray_release.result import ExitCode
 
 
-class RunScriptTest(unittest.TestCase):
-    def setUp(self) -> None:
-        self.tempdir = tempfile.mkdtemp()
-        self.state_file = os.path.join(self.tempdir, "state.txt")
-        self.test_script = os.path.join(
-            os.path.dirname(__file__), "..", "..", "run_release_test.sh"
-        )
+@pytest.fixture
+def setup(tmpdir):
```

Comment:
Did chatGPT come up with this fixture as well 😱

Reply:
Yep :-)